src/components/appNavigation/AppNavigation.tsx

```diff
@@ -0,0 +1,78 @@
+import React, { useEffect } from "react";
+import "./AppNavigation.css";
+import { useState } from "react";
+import { useLocation } from "react-router-dom";
+import { IconButton, Link, Typography } from "@mui/material";
+import contacts from "../../assets/images/appNavigation/contacts.png";
+import contactsRed from "../../assets/images/appNavigation/contactsRed.png";
+import homeHealth from "../../assets/images/appNavigation/home_health.png";
+import homeHealthRed from "../../assets/images/appNavigation/home_healthred.png";
+import prescription from "../../assets/images/appNavigation/prescriptions.png";
+import prescriptionRed from "../../assets/images/appNavigation/prescriptionsRed.png";
+
+const navItems = [
+  {
+    path: "/medications",
+    name: "Medications",
+    icon: homeHealth,
+    activeIcon: homeHealthRed,
+  },
+  {
+    path: "/therapies",
+    name: "Therapies",
+    icon: prescription,
+    activeIcon: prescriptionRed,
+  },
+  {
+    path: "/contacts",
+    name: "Contacts",
+    icon: contacts,
+    activeIcon: contactsRed,
+  },
+];
+
+export function AppNavigation() {
+  const location = useLocation();
+
+  if (location.pathname === "/login") {
+    return null;
+  }
+
+  const [activepage, setActivePage] = useState(location.pathname);
+
+  useEffect(() => {
+    setActivePage(location.pathname);
+  }, [location.pathname]);
+
+  return (
+    <div className="mytherapie">
+      <div className="menuBot">
+        {navItems.map(({ path, name, icon, activeIcon }) => (
```

Comment:
use Interface `NaviagtionItem ` that you have create inside `navigationData.ts` here